tests/Plugins/__snapshots__/GitHub_EndToEndTest.ambr

```diff
@@ -871,13 +870,13 @@
   │                                                                                                                      │
   │ [31m╭─[0m[31m [Error] RequireSignedCommitsRule [0m[31m──────────────────────────────────────────────────────────────────────────────[0m[31m─╮[0m │
   │ [31m│[0m                                                                                                                  [31m│[0m │
-  │ [31m│[0m Validates that 'Require signed commits' is set to the expected value.                                            [31m│[0m │
+  │ [31m│[0m Validates that 'Require signed commits' is set to True.                                                          [31m│[0m │
```

Comment:
This is confusing to me. If I recall correctly, the default behavior for `RequireSignedCommitsRule` is to validate the the setting is enabed/checked/true. However, in this scenario, we have toggled the default and now want to validate that the setting is disabled/unchecked/false.   
 

The statement on line 873 seems to be describing the default behavior, while the line in 875 is saying "The setting is not set to the value that you wanted to validate." and the statement on 879 is explaining what is necessary to make the error on line 875 go away.  
  
In this context, the line on 873 seems to be out of place in a subtle way.

Reply:
This was a good catch. I found a bug in the way I was formatting the `description` string, which is now fixed.